context/app/static/js/components/Detail/Visualization/Visualization.jsx

```diff
@@ -60,7 +60,9 @@ function Visualization(props) {
   return (
     <StyledSectionContainer id="visualization">
       <StyledHeader>
-        <StyledHeaderText>Visualization</StyledHeaderText>
+        <StyledHeaderText variant="h4" component="h2">
```

Comment:
There was a PR yesterday, I think, that got rid of a lot of these attribute usages, and I'm reluctant to go in the other direction. I've wondered if we could just have an `<H1>`, `<H2>`, that would wrap `<h1>` with whatever material UI details we need.

Reply:
I think I fixed that, let me know if the change addresses your concerns.